modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js

```diff
@@ -986,6 +986,12 @@
 			} else if ( pageCanLoadEditor ) {
 				// Allow instant switching to edit mode, without refresh
 				$caVeEdit.off( '.ve-target' ).on( 'click.ve-target', init.onEditTabClick.bind( init, 'visual' ) );
+				$caVeEdit.on( 'mouseover.ve-target', function () {
```

Comment:
Let's demark our changes to this file with `// Fandom change` comments for easier reconciliation later

Reply:
Added